src/SonarScanner.MSBuild.TFS.Classic/Resources.resx

```diff
@@ -146,7 +146,7 @@
     <value>Visual Studio setup configuration was not found.</value>
   </data>
   <data name="CONV_ERROR_ConversionToolFailed" xml:space="preserve">
-    <value>Failed to convert the downloaded code coverage tool to XML. No code coverage information will be uploaded to SonarQube.
+    <value>Failed to convert the binary code coverage reports to XML. No code coverage information will be uploaded to the server (SonarQube/SonarCloud).
```

Comment:
is there a way to test this?

Reply:
Theoretically yes. We would have to move to the new branch to avoid conflicts (or implementing the test twice) and refactor the code by extracting the `CoverageFileUtility` in another class that we can mock. 

I would say it's too much of an effort for a message change.

Reply:
yes, we have a test on the feature branch. We will have to update it when doing the rebase: https://github.com/SonarSource/sonar-scanner-msbuild/blob/feature/BreakingChanges/Tests/SonarScanner.MSBuild.TFS.Test/Classic/BinaryToXmlCoverageReportConverterTests.cs#L168